src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuffer binaryValue = new StringBuffer();
```

Comment:
What is the difference between StringBuffer and StringBuilder?

Reply:
StringBuffer and StringBuilder are the same. But StringBuilder are not synchronized and works faster.

Reply:
explain please)
where is my mistake?

Reply:
https://www.journaldev.com/538/string-vs-stringbuffer-vs-stringbuilder
"...StringBuffer provides Thread safety but on a performance cost..."